web-client/public/js/graph.js

```diff
@@ -654,27 +653,27 @@ export var drawGraph = function (network) {
 
     if (network.sheetType === "weighted") {
         link.append("text")
-        .attr("class", "weight")
-        .attr("text-anchor", "middle")
-        .attr("text-anchor", "middle")
-        .text(function (d) {
-            return d.value.toPrecision(4);
-        });
+            .attr("class", "weight")
+            .attr("text-anchor", "middle")
+            .attr("text-anchor", "middle")
+            .text(function (d) {
+                return d.value.toPrecision(4);
+            });
 
         weight = weight.data(network.links)
-        .enter().append("text")
-        .attr("class", "weight")
-        .attr("text-anchor", "middle")
-        .text(function (d) {
-            return d.value.toPrecision(4);
-        })
-        .each(function (d) {
-            d.weightElement = d3.select(this);
-        });
+            .enter().append("text")
+            .attr("class", "weight")
+            .attr("text-anchor", "middle")
+            .text(function (d) {
+                return d.value.toPrecision(4);
+            })
+            .each(function (d) {
+                d.weightElement = d3.select(this);
+            });
 
     }
 
-  /* Big thanks to the following for the smart edges
+    /* Big thanks to the following for the smart edges
```

Comment:
I see a lot of re-indenting in the latest changes to this PR and most of them look right—but this one doesn’t. Note where the asterisks are in the succeeding lines.
```suggestion
  /* Big thanks to the following for the smart edges
```